### PR TITLE
bugfix #62 & modify method when parse image

### PR DESCRIPTION
**Pull request by SarahhhC**
1. bug fix in travis failure when deploy mode.

2. method changed when parse image (formidable -> base64) due to session issue.

3. other refactoring.

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -212,102 +211,83 @@ export function getJobCategory(req, res, next) {
...
-          let bucketName = 'yodabucket';
-          let imageKey = `profile/${req.session._id}.png`;
-          let readStream = fs.createReadStream(file.path);
+          throw new Error(userCallback.ERR_MONGOOSE);
+        }
+      })
+      .then(keyData => {
+        if (keyData) {
+          if (req.body.image == null) {
+            console.log(req.body.image);
```
Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: no console

File: src/server/controllers/users.controller.js [hunk 2]
```diff
@@ -212,102 +211,83 @@ export function getJobCategory(req, res, next) {
...
+            res.status(200).json({ msg: userCallback.SUCCESS_UPDATE_WITHOUT_IMAGE });
+          } else {
+            console.log(req.body.image);
+            let bucketName = 'yodabucket';
+            let now = new Date();
+            let imageKey = `profile/${req.session._id}/${now.getTime()}/.png`;
```
Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: Is this file name for the image?
I think '_' would be more readable than '/'

File: src/server/controllers/users.controller.js [hunk 3]
```diff
@@ -212,102 +211,83 @@ export function getJobCategory(req, res, next) {
...
+          throw new Error(userCallback.ERR_MONGOOSE);
+        }
+      })
+      .then(keyData => {
+        if (keyData) {
+          if (req.body.image == null) {
+            console.log(req.body.image);
+            res.status(200).json({ msg: userCallback.SUCCESS_UPDATE_WITHOUT_IMAGE });
+          } else {
+            console.log(req.body.image);
```
Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: erase~

File: src/server/controllers/users.controller.js (hunk 1 above)
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: yup!

File: src/server/controllers/users.controller.js (hunk 3 above)
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: ok~~

File: src/server/controllers/users.controller.js (hunk 2 above)
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: We make each user profile folder with their id and each folder contains images named with timestamp. Maybe using '_' would be good also, but previous one would be better to manage lots of file. Thanks!

**Comment by KyuaKwon** [comment]
Let's stare the Travis!
:shipit: 👀

**Pull request merged**